### PR TITLE
Swope known issue (3)

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Comment by elgnay** [comment]
@swopebe I like the idea to split them into two issues. For the second one, `_appliedmanifestwork_ not removed and agentID is missing in the specification`, it will not happen when both primary and restore hub is 2.6 hub. In another world, the appliedmanifestworks will be leftover only when the primary hub is 2.6 hub and restore hub is 2.7/2.8/2.9 hub.

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -894,6 +889,63 @@ You can also delete the remaining `appliedmanifestwork` manually by running the
...
+----
+
+You can also delete the remaining `appliedmanifestwork` manually by running the following command:
+
+----
+oc delete appliedmanifestwork <the-left-appliedmanifestwork-name>
+----
+
+[#appliedmanifest-agentid-missing]
+=== _appliedmanifestwork_ not removed and agentID is missing in the specification
```
Comment (oafischer) [inline release_notes/known_issues.adoc]: agentID has backticks in the paragraph, do we want to make it italic in the title to match?

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -894,6 +889,63 @@ You can also delete the remaining `appliedmanifestwork` manually by running the
...
+//2.7+:ACM-7588
+
+When you are using {product-title-short} 2.6 as your primary hub cluster, the `agentID` is missing in the specification of `appliedmanifestworks` because the field is introduced in the 2.7 release. This results in the extra `appliedmanifestworks` for the primary hub on the managed cluster.
+
+To avoid the issue, upgrade the primary hub cluster to {product-title-short} 2.7, then restore the backup on a new hub cluster.
+
+Fix the managed clusters by setting the `spec.agentID` manually for each `appliedmanifestwork`.
+
+. Run the following command to get the agent ID:
```
Comment (oafischer) [inline release_notes/known_issues.adoc]: Does agentID need the space in this line?

**Review by oafischer — APPROVED** [review]
2 comments, otherwise looks great

**Review by oafischer — APPROVED** [review]

**Pull request merged**